src/api/api.ts

```diff
@@ -124,6 +146,85 @@ class API {
     }
   }
 
+  async getChatRoomWithGivenUser(
+    participantEmail: string,
+    acceptTimezone: string | null
+  ): Promise<ChatRoom> {
+    try {
+      const headers: HeadersInit = {
+        "Accept-Role": store.user.profile!.selected_profile!.toUpperCase(),
+      };
+      if (acceptTimezone !== null) {
```

Comment:
`acceptTimezone` ma typ `string | null`, więc skróciłbym:
```suggestion
      if (acceptTimezone) {
```